src/Lucene.Net/Search/DefaultSimilarity.cs

```diff
@@ -105,4 +107,34 @@ public virtual bool DiscountOverlaps
 	        set { internalDiscountOverlaps = value; }
 	    }
 	}
+
+    /// <summary>Expert: light weight scoring implementation that disables idfExplanation. </summary>
+    [Serializable]
+    public class LightWeightSimilarity : DefaultSimilarity
+    {
+        public override Explanation.IDFExplanation IdfExplain(ICollection<Term> terms, Searcher searcher, IState state)
+        {
+            return TheExplanation;
+        }
+
+        public override Explanation.IDFExplanation IdfExplain(Term term, Searcher searcher, IState state)
+        {
+            return TheExplanation;
+        }
+
+        private static readonly LightWeightSimilarity _instance = new LightWeightSimilarity();
+        public static Similarity Instance => _instance;
```

Comment:
can we now just simplify this to `public static readonly Similarity Instance = new LightWeightSimilarity();` ?

Reply:
yes